snap/info.go

```diff
@@ -357,12 +357,21 @@ func infoFromSnapYamlWithSideInfo(meta []byte, si *SideInfo) (*Info, error) {
 	return info, nil
 }
 
+type SnapVanishedError struct {
+	Snap     string
+	Revision Revision
+}
+
+func (e SnapVanishedError) Error() string {
+	return fmt.Sprintf("cannot find mounted snap %q at revision %s", e.Snap, e.Revision)
```

Comment:
s/mounted/installed/.. that's an implementation detail leaking into the UI.. the user has no idea about the fact we mount it.